pom.xml

```diff
@@ -81,6 +82,18 @@
       <artifactId>native-lib-loader</artifactId>
       <version>${dependency.nativelibloader.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>${dependency.log4j.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
With log4j-core adding api explicitly is not required. 
We may want to add this however:
```
    <dependency>
      <groupId>org.apache.logging.log4j</groupId>
      <artifactId>log4j-slf4j-impl</artifactId>
      <version>${dependency.log4j.version}</version>
      <scope>test</scope>
    </dependency>
 ```

As a replacement for `logback-classic`. That's because our dependency is already logging to slfj4, so we may want to consume those log our way (i.e. apply our formatting to their logs) - `log4j-core` does not do that by itself. Logback classic would, but - hey - xml config :)

Reply:
I think this is a good idea, it will make the native-lib-loader logs a bit easier on the eyes.  @pietrygamat Do you mind adding it?  As I'm sure you can see, I'm not overly familiar with configuring logs, any help is appreciated.  Please feel free to push directly to this branch.